### PR TITLE
V0.2.4

### PR DESCRIPTION
**Pull request by hansvancalster**
Fixes checklist related stuff

### CODE REVIEW

**Comment by hansvancalster** [comment]
check_package failed because  

```
Missing repository secret(s) ORCID_TOKEN, CODECOV_TOKEN on GitHub.
See https://inbo.github.io/checklist/articles/getting_started.html#online-setup-2 for more details.
```
but these tokens have been added to the repo. @ElsLommelen any idea what it going on?

**Comment by hansvancalster** [comment]
@ElsLommelen I've renewed the tokens, but with no avail. @ThierryO anything else you can think of that I'm doing wrong?

**Comment by ThierryO** [comment]
The GHA workflow files were outdated. I've updated them using `checklist::setup_package()` and then commit the relevant changes.

**Comment by hansvancalster** [comment]
Ah that will probably be the cause. Thank you! Is this documented?

**Comment by ThierryO** [comment]
It is not documented. It will only affect packages that used an early version of checklist. I maintain most of them. The other maintainers will contact me. Therefore I think it might be more confusing to add this to the documentation.

File: R/clean_git.R [hunk 1]
```diff
@@ -140,7 +140,7 @@ clean_git <- function(repo =  ".", verbose = TRUE) {
   ) {
     git_branch_create(
       branch = main_branch,
-      ref = paste("refs/remotes/origin", main_branch, sep = "/"),
+      ref = paste("refs/remotes/origin", main_branch, sep = "/"), # nolint: nonportable_path_linter, line_length_linter.
```
Comment (ThierryO) [inline R/clean_git.R:143]: You could also write it like this.

```suggestion
      ref = paste("refs", "remotes", "origin", main_branch, sep = "/"),
```

**Comment by ElsLommelen** [comment]
Eh, up to you to decide.  If you want to add these changes to the main within a week, then you better don't wait, I'm afraid.  If you're not hurried to have it merged, it is easy for me if you leave it open.  ;-)

**Comment by hansvancalster** [comment]
I'll leave it open then

**Comment by ElsLommelen** [comment]
Eh, why do you close and delete this branch without merging?
Also: don't forget to implement the `protocolcheck` class to function `check_frontmatter()`.  (I can help with it, but I leave it up to you to let me know when you need/want help.)

**Comment by hansvancalster** [comment]
Please see https://github.com/inbo/protocolhelper/pull/71

**Pull request closed without merging**